boards/aitewinrobot-esp32c3-supermini/board.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "Aitewin Robot ESP32-C3 SuperMini",
+  "version": 1.8,
```

Comment:
version should be an integer - set it to 1, since this is the first version that will be released.